test_aks.py

```diff
@@ -31,10 +37,52 @@ def cluster_labels():
     return config.items('cluster_labels')
 
 
-def test_simpleFlow(login, variables_values, cluster_labels):
+def setup_function():
+    global terraform_cmd, variables_values, logger
+    logger.info("Start setup")
+    variables_values = {
+        'subscription_id': os.environ.get('ARM_SUBSCRIPTION_ID'),
+        "location": os.environ.get('ARM_TEST_LOCATION'),
+        "client_id": os.environ.get('ARM_CLIENT_ID'),
+        "client_secret": os.environ.get('ARM_CLIENT_SECRET'),
+        "tenant_id": os.environ.get('ARM_TENANT_ID'),
+        "ssh_username": 'openqa',
+        "cluster_admin_password": str(uuid.uuid4()),
+        "uaa_admin_client_secret": str(uuid.uuid4()),
+        "dns_zone_name": "openqa.com",
+        "cap_domain": "cap.openqa.com",
+        "email": "akstest@suse.com"}
+    if "AKS_TEST_SSH_PULIC_KEY" in os.environ:
+        variables_values["ssh_public_key"] = os.environ.get(
+            'AKS_TEST_SSH_PULIC_KEY')
+    else:
+        variables_values["ssh_public_key"] = \
+            "/home/asmorodskyi/.ssh/id_rsa_pc.pub"
```

Comment:
Let's not have special `asmorodskyi` stuff there :)
```suggestion
            "{}/.ssh/id_rsa_pc.pub".format(os.environ.get('HOME'))
```

Reply:
oops , this was actually suppose to be removed before commit . About with what replace defaults - keeping in mind that at some point of time we plan to run this in openQA I think it make sense to have same defaults https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/provider.pm#L169